recipes/llvmdev/meta.yaml

```diff
@@ -0,0 +1,68 @@
+# (Mostly) blatant copy of https://github.com/numba/llvmlite/tree/master/conda-recipes/llvmdev
+
+{% set shortversion = "3.8" %}
+{% set version = "3.8.1" %}
+{% set sha256 = "6e82ce4adb54ff3afc18053d6981b6aed1406751b8742582ed50f04b5ab475f9" %}
+
+package:
+  name: llvmdev
+  version: {{ version }}
+
+source:
+  fn: llvm-{{ version }}.src.tar.xz
+  url: http://llvm.org/releases/{{ version }}/llvm-{{ version }}.src.tar.xz
+  sha256: {{ sha256 }}
+  patches:
+    # https://llvm.org/bugs/show_bug.cgi?id=15356
+    - llvm-elf-relocation.patch
+    # https://llvm.org/bugs/show_bug.cgi?id=25788
+    - llvm-parse-fp-constants.patch
+    # https://llvm.org/bugs/show_bug.cgi?id=25565
+    - llvm-build-race-condition.patch
+    # http://lists.llvm.org/pipermail/llvm-dev/2016-January/094520.html
+    - llvm-lto-static.patch  # [win]
+
+build:
+  number: 0
+  skip: true  # [(unix and py3k) or (win and py<35)]
+  features:   # [win and py35]
+    - vc14    # [win and py35]
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    # Needed to unpack the source tarball
+    - m2-xz  # [py27 and win32]
+    - m2w64-xz  # [py27 and win64]
+    # Needed to build LLVM
+    - python
+    - system  # [linux and not armv7l]
+  run:
+    - system  # [linux and not armv7l]
+
+test:
+  requires:                                                  # [win]
+    - python {{ environ['PY_VER'] + '*' }}                   # [win]
+
+  commands:
+    - $PREFIX/bin/llvm-config --libs                         # [not win]
+    - $PREFIX/bin/llc -version                               # [not win]
+
+    - if not exist %LIBRARY_INC%\\include\\llvm\\Pass.h exit 1     # [win]
+
+    - test -f $PREFIX/include/llvm/Pass.h                    # [unix]
+    - test -f $PREFIX/lib/libLLVMCore.a                      # [not win]
+
+about:
+  home: http://llvm.org/
+  license: NCSA
+  license_file: LICENSE.TXT
+  summary: Development headers and libraries for LLVM
+
+extra:
+  recipe-maintainers:
+    - inducer
+    - pitrou
+    - jakirkham
```

Comment:
Not sure I need to be a maintainer, but definitely plan to keep an eye on this feedstock.